app/lib/LevelSetupManager.coffee

```diff
@@ -67,10 +67,11 @@ module.exports = class LevelSetupManager extends CocoClass
     if @level.usesConfiguredMultiplayerHero()
      @onInventoryModalPlayClicked()
      return
-
-    if @level.isType('course', 'course-ladder', 'game-dev', 'web-dev') or window.serverConfig.picoCTF
-      @onInventoryModalPlayClicked()
-      return
+  
+    if @level.isType('course','course-ladder', 'game-dev', 'web-dev') or window.serverConfig.picoCTF
+      if @level.isType('course') and not me.showHeroAndInventoryModals()
+        @onInventoryModalPlayClicked()
+        return
```

Comment:
Just realized the above if condition is not framed correctly. It won't work as intended for level types 'course-ladder', 'game-dev', 'web-dev'. I need to fix it.